src/towncrier/_writer.py

```diff
@@ -24,7 +24,7 @@ def append_to_newsfile(
         else:
             with io.open(news_file, "r", encoding="utf8") as f:
                 existing_content = f.read()
-        existing_content = existing_content.split(start_line, 1)
+        existing_content = existing_content.split(start_string, 1)
     else:
         existing_content = [u""]
```

Comment:
```suggestion
        existing_content = [u"", u""]
```

Reply:
Hmm, or maybe not since the other code can also result in either a single or double element list depending...  Time to think a bit more.